test/parallel/test-child-process-cwd.js

```diff
@@ -66,10 +68,27 @@ function testCwd(options, expectPidType, expectCode = 0, expectData) {
     }));
 }
 
+{
+  assert.throws(() => {
+    testCwd({
+      cwd: new URL(`http://${tmpdir.path}`),
+    }, 'number', 0, tmpdir.path);
+  }, /The URL must be of scheme file/);
+
+  if (process.platform !== 'win32') {
+    assert.throws(() => {
+      testCwd({
+        cwd: new URL(`file://host${tmpdir.path}`),
```

Comment:
There's no guarantee that `tmpdir.path` would make a valid URL:
```suggestion
        cwd: new URL(`file://host/dev/null`),
```

Reply:
Hmmmm, because tmpdir.path used for a valid test case above:

```js
testCwd({ cwd: tmpdir.path }, 'number', 0, tmpdir.path);
```